src/core/server/saved_objects/import/extract_errors.ts

```diff
@@ -34,17 +34,16 @@ export function extractErrors(
       const originalSavedObject = originalSavedObjectsMap.get(
         `${savedObject.type}:${savedObject.id}`
       );
-      const title =
-        originalSavedObject &&
-        originalSavedObject.attributes &&
-        originalSavedObject.attributes.title;
+      const title = originalSavedObject?.attributes?.title;
+      const { newId } = savedObject;
       if (savedObject.error.statusCode === 409) {
         errors.push({
           id: savedObject.id,
           type: savedObject.type,
           title,
           error: {
             type: 'conflict',
+            ...(newId && { destinationId: newId }),
```

Comment:
If an object was successfully created with a different ID than what was specified in the import objects, that new ID is returned to the consumer as the `newId` field (in the `successResults` array).

If an object was NOT successfully created and it was an "inexact match" conflict, that ID is returned to the consumer as the `destinationId` field. in the conflict error (in the `errors` array).

At the time I was writing this I thought it made sense not to conflate the two. But they are really describing the same thing. Maybe we should rename `newId` in the `successResults` to `destinationId`? Thoughts?